projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.ts

```diff
@@ -71,8 +76,15 @@ export class FhirDataService {
       return teardownLogic;
     });
   }
-  addPatientData(reportBPValue: any) {
-    const resource = {
+
+  addPatientData(resource: any, requestOptions: {}) {
```

Comment:
A parameter of type `{}` (empty object) seems useless. Did you mean to use `{}` as the default value instead?

Reply:
okay, I will pass only resource in addPatientData.
![image](https://user-images.githubusercontent.com/117890382/217468514-ef6643a4-0990-4096-9c76-ee753d11496f.png)